packages/fluentui/react-northstar/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -13,7 +13,7 @@ const popupContentStyles: ComponentSlotStylesPrepared<PopupContentStylesProps, P
       pointerEvents: 'none',
       ...getContainerStyles({
         placement: p.basePlacement,
-        margin: v.pointerMargin,
+        padding: v.pointerMargin,
```

Comment:
It's actually padding. Decided to keep variable's name to avoid breaking changes